build.gradle

```diff
@@ -1,29 +1,16 @@
 apply plugin: 'dev.equo.ide'
+
 equoIde {
 	branding().title('Spotless').icon(file('_images/spotless_logo.png'))
 	welcome().openUrl('https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md')
 	gradleBuildship().autoImport('.')
 }
 
-repositories {
-	mavenCentral()
```

Comment:
i know spot needs this also rewrite.

Is it working without, why so?

Actually would expect mavenCentral to be kind of default best practice active by default following convention principle.